### ISSUE TITLE
"Introduction to the DOM" Link color doesn't change in example if text color isn't red 

### ISSUE THREAD

**Issue by MLarik** [mdn/content#20591]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Introduction

### What information was incorrect, unhelpful, or incomplete?

When I change color of link to green, before I change text color - it doesn't change
When I change text color to red first and then change link color - it changes.
Why?

### What did you expect to see?

link color changes even if I change it immediately

### Do you have any supporting links, references, or citations?

![image](https://user-images.githubusercontent.com/32640722/189664292-f804e28b-ed57-46b6-b286-c784ef57adc4.png)
![image](https://user-images.githubusercontent.com/32640722/189664707-5ab6859a-2da3-44f5-af9c-03691c1f3803.png)

### Do you have anything more you want to share?

no

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/document_object_model/introduction`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Introduction
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/document_object_model/introduction/index.md
* Last commit: https://github.com/mdn/content/commit/1511e914c6b1ce6f88056bfefd48a6aa585cebce
* Document last modified: 2022-09-09T05:10:34.000Z

</details>

**Comment by Abhiram-Gullapalli** [comment]
Yep it's exhibiting a strange behavior, would like to work on this.

**Comment by wbamberg** [comment]
I think this is a duplicate of https://github.com/mdn/content/issues/12415.

There are several problems with this example:
* it includes the HTML as a complete page, whereas live samples should only include the contents of the `<body>`.
* it has the script inline in the `<head>`, which makes the example hard to read and isn't good practice.
* it uses HTML attributes for event handlers, which is bad practice.
* it uses the work "simple" which is almost always not OK.
* as @hamishwillee points out in https://github.com/mdn/content/issues/12415, setting styles directly in HTML isn't usually good practice either. Hamish had some good suggestions in https://github.com/mdn/content/issues/12415#issuecomment-1080142210 .

So basically the example needs to be rewritten. @Abhiram-Gullapalli , let us know if you would like to do this, or I might have a go.
